contributors_guide.rst

```diff
@@ -0,0 +1,376 @@
+3-Clause BSD license. Py-ART is used by the 
+`Atmospheric Radiation Measurement (ARM) Climate Research Facility 
+<http://www.arm.gov>`_ for working with data from a number of precipitation
+and cloud radars, but has been designed so that it can be used by others in
+the radar and atmospheric communities to examine, processes, and analyze
+data from many types of weather radars. 
+
+
+Important Links
+---------------
+
+- Official source code repository: https://github.com/ARM-DOE/pyart
+- HTML documentation: http://arm-doe.github.io/pyart-docs-travis/
+- Examples: http://arm-doe.github.io/pyart/dev/auto_examples/index.html
+- Mailing List: http://groups.google.com/group/pyart-users/
+- Issue Tracker: https://github.com/ARM-DOE/pyart/issues
+
+
+Citing
+------
+
+If you use the Python ARM Radar Toolkit (Py-ART) to prepare a publication
+please cite:
+
+    Helmus, J.J. & Collis, S.M., (2016). The Python ARM Radar Toolkit
+    (Py-ART), a Library for Working with Weather Radar Data in the Python
+    Programming Language. Journal of Open Research Software. 4(1), p.e25.
+    DOI: http://doi.org/10.5334/jors.119
+
+Py-ART implements many published scientific methods which should *also* be
+cited if you make use of them.  Refer to the **References** section in the
+documentation of the functions used for information on these citations.
+
+
+Install
+-------
+
+The easiest method for installing Py-ART is to use the conda packages from
+the latest release.  To do this you must download and install 
+`Anaconda <http://continuum.io/downloads>`_ or 
+`Miniconda <http://continuum.io/downloads>`_.  
+Then use the following command in a terminal or command prompt to install
+the latest version of Py-ART::
+
+    conda install -c conda-forge arm_pyart
+
+To update an older version of Py-ART to the latest release use::
+
+    conda update -c conda-forge arm_pyart
+
+Resources
+=========
+
+Pyart:
+
+- https://github.com/openradar/AMS-Short-Course-on-Open-Source-Radar-Software
+- https://github.com/EVS-ATMOS/pyart_short_course
+- https://www.youtube.com/watch?v=diiP-Q3bKZw
+- http://arm-doe.github.io/pyart/dev/auto_examples/index.html
+
+Git:
+
+- https://git-scm.com/book/en/v2
+- https://docs.scipy.org/doc/numpy/dev/gitwash/development_workflow.html
+
+
+Proper Code Style
+=================
+
+Py-ART follows pep8 coding standards. To make sure your code follows the pep8
+style, you can use a variety of modules that can check for you. Two popular
+pep8 check modules are pycodestyle and pylint.
+
+For more on pep8 style:
+
+- https://www.python.org/dev/peps/pep-0008/
+
+To install pycodestyle::
+
+        pip install pycodestyle
+
+To use pycodestyle::
+
+        pycodestyle filename
+
+pylint is also useful to check for pep8 compliance and should already be
+installed with python 3.5.
+
+To get a detailed pylint report::
+
+        pylint filename
+
+If you want to just see what line number and the issue, just use::
+
+        pylint -r n filename
+
```

Comment:
Links to the documentation of pycodestyle and pylint with some text about how both can be configured by users would be appropriate here.

https://pycodestyle.readthedocs.io/en/latest/
https://www.pylint.org/